staging/src/k8s.io/cli-runtime/pkg/printers/json_test.go

```diff
@@ -138,3 +140,75 @@ func TestPrintersSuccess(t *testing.T) {
 		}
 	}
 }
+
+func TestStripManagedFields(t *testing.T) {
```

Comment:
I'm curious if "diff" will work the same way. i.e. will it diff the managed fields or will they be dropped anyway? Would you mind double checking how we print objects for diff?

Reply:
I have tried locally and it turns out that `diff` will diff the managed fields as well.